docs/maui/views/Popup.md

```diff
@@ -58,30 +60,28 @@ public partial class SimplePopup : Popup
 
 ### C#
 
+The following section covers how to create a `Popup` using C#.
+
 ```csharp
 using CommunityToolkit.Maui.Views;
 
-var popup = new Popup
+var popup = new ContentView
 {
-    Content = new VerticalStackLayout
+    Content = new Label
     {
-        Children = 
-        {
-            new Label
-            {
-                Text = "This is a very important message!"
-            }
-        }
+        Text = "This is a very important message!"
     }
 };
 ```
 
 ## Presenting a Popup
 
-Once the `Popup` has been built it can then be presented through the use of our `Popup` extension methods or through the [`IPopupService`](popup-service.md) implementation from this toolkit.
+Once the `Popup` has been built it can then be presented through the use of the `Popup` extension methods or through the [`IPopupService`](popup-service.md) implementation from this toolkit.
```

Comment:
Where should I use the extension methods? In a page, application, the popup itself?